lib/rules/selector-combinator-space-after/__tests__/index.js

```diff
@@ -415,8 +518,61 @@ testRule(rule, {
   reject: [
     {
       code: "a+  a {}",
+      fixed: "a+ a {}",
       description: "two spaces after + combinator",
       message: messages.expectedAfter("+")
     }
   ]
 });
+testRule(rule, {
+  ruleName,
+  syntax: "scss",
+  config: ["always"],
+  fix: true,
+
+  accept: [
+    {
+      code: ".a { &.b {} }",
+      description: "ignore constructs"
+    },
+    {
+      code: ".a { & .b {} }",
+      description: "ignore constructs"
+    }
+  ],
+
+  reject: [
+    {
+      code: ".a { &+.b {} }",
+      fixed: ".a { &+ .b {} }",
+      message: messages.expectedAfter("+")
+    }
+  ]
+});
+
+// cssnext
+testRule(rule, {
```

Comment:
Can you move the tests in this block into the first `testRule` block (the one without any `syntax: property`), and then remove this `testRule` block, please?

The syntax [is valid draft spec](https://tabatkins.github.io/specs/css-nesting/#direct) so [belongs](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#best-practices) in the first `testRule`.